src/android/BasePrint.java

```diff
@@ -145,6 +145,15 @@ private void getPreferences() {
                             .parseBoolean(sharedPreferences.getString("endCut", new Boolean(true).toString()));
                     break;
 
+                case QL_1110NWB:
+                mPrinterInfo.labelNameIndex = LabelInfo.QL1100.valueOf(
+                        sharedPreferences.getString("paperSize", LabelInfo.QL1100.W103H164.toString())).ordinal();
```

Comment:
Thank you very much for this PR @LimitTech 
Most appreciated.
Would you be kind enough to add this new label to the list at 
https://github.com/MenelicSoftware/cordova-plugin-brother-label-printer/blob/master/www/printer.js#L46

please